README.md

```diff
@@ -917,6 +917,110 @@ class TritonPythonModel:
 A complete example for sync and async BLS in Python backend is included in the
 [Examples](#examples) section.
 
+Starting from 23.02 release, you can execute inference requests on decoupled
+models in both [default mode](#default-mode) and
+[decoupled mode](#decoupled-mode). By using function `stream_exec`, you can get
+a list of responses returned by a decouple model. Example below shows how to use
+this feature:
+
+```python
+import triton_python_backend_utils as pb_utils
+
+
+class TritonPythonModel:
+  ...
+    def execute(self, requests):
+      ...
+      # Create an InferenceRequest object. `model_name`,
+      # `requested_output_names`, and `inputs` are the required arguments and
+      # must be provided when constructing an InferenceRequest object. Make sure
+      # to replace `inputs` argument with a list of `pb_utils.Tensor` objects.
+      inference_request = pb_utils.InferenceRequest(
+          model_name='model_name',
+          requested_output_names=['REQUESTED_OUTPUT_1', 'REQUESTED_OUTPUT_2'],
+          inputs=[<pb_utils.Tensor object>])
+
+      # `pb_utils.InferenceRequest` supports request_id, correlation_id, and model
+      # version in addition to the arguments described above. These arguments
+      # are optional. An example containing all the arguments:
+      # inference_request = pb_utils.InferenceRequest(model_name='model_name',
+      #   requested_output_names=['REQUESTED_OUTPUT_1', 'REQUESTED_OUTPUT_2'],
+      #   inputs=[<list of pb_utils.Tensor objects>],
+      #   request_id="1", correlation_id=4, model_version=1, flags=0)
+
+      # Execute the inference_request and wait for the response
+      inference_responses = inference_request.stream_exec()
+
+      for inference_response in inference_responses:
+        # Check if the inference response has an error
+        if inference_response.has_error():
+            raise pb_utils.TritonModelException(inference_response.error().message())
+        else:
+            # Extract the output tensors from the inference response.
+            output1 = pb_utils.get_output_tensor_by_name(inference_response, 'REQUESTED_OUTPUT_1')
```

Comment:
Will fix the character count. Would we want to do the same for the code section? I think the code section is easier to read as it is. Let me know if you think otherwise,

Reply:
Yes, it would be great to keep the character count below 80 in the code too.

Reply:
Updated.